scipp_widgets/widgets.py

```diff
@@ -188,3 +187,37 @@ def _process(self, kwargs):
         output = self.callable(**kwargs)
         self.scope[output_name] = output
         display(self.scope[output_name])
+
+
+class LoadWidget(WidgetBase):
+    """
+    Provides a simple graphical wrapper around a load function,
+    adding the return value to the notebooks scope labelled
+    by file name.
+    """
+    def __init__(self,
+                 wrapped_func: Callable,
+                 inputs: Iterable[IInput],
+                 button_name: str = 'Load',
+                 layout='row wrap',
+                 filename_param='filename',
+                 hide_code: bool = False):
+        """
+        :param filename_param: Name of input parameter to use as file name.
+        """
```

Comment:
Not quite sure I understand the param. Is this the name of the param passed to `wrapped_func`? Or the name of the output variable added to the notebook scope?